forloop_modules/redis/config/config.py

```diff
@@ -14,14 +14,14 @@ class Config:
 
     # None values for Host and Password are correct options:
     # in-memory local caching will be used instead of a Redis server
-    FORLOOP_REDIS_HOST: Optional[str] = None # "localhost" #localhost doesn't work for devs with DummyKVRedis
+    FORLOOP_REDIS_HOST: Optional[str] = 'localhost' # localhost doesn't work for devs with DummyKVRedis
```

Comment:
are you ignoring the comment?

Reply:
Not at all, accidentally committed my local `config.py`